official/.gitignore

```diff
@@ -0,0 +1 @@
+tmp
```

Comment:
I think it would be less confusing to rather add `official/tmp` in the top-level `.gitignore`.

Reply:
+1